netlogo-gui/src/main/lab/gui/LabManager.scala

```diff
@@ -36,7 +36,7 @@ class LabManager(val workspace:        GUIWorkspace,
     protocols.clear()
   }
   private lazy val dialog = new ManagerDialog(this, dialogFactory, menuFactory)
-  def show() { dialog.update(); dialog.setVisible(true) }
+  def show() { dialog.update(); dialog.setVisible(true); dialog.setLocationRelativeTo(workspace.getFrame) }
```

Comment:
Please split this method out into multiple lines, as it's getting a bit too long for one line. When splitting into multiple lines, you can also remove the semicolons.